src/components/Aframe/my-loader.js

```diff
@@ -574,7 +562,16 @@ AFRAME.registerComponent("myloader", {
     }
 
     if (oldData.volumeData !== this.data.volumeData) {
-      this.loadModel();
+		var parent_folder = this.data.volumeData.substr(0,this.data.volumeData.lastIndexOf("/")+1)
+	    
+		fetch(this.data.volumeData).then(res => res.json())
+		.then(jsonData => {
+			// Do something with your data
+			// var fullFilePath = parent_folder + jsonData['fileName']
+			// console.log("fullFilePath: "+fullFilePath)
+			this.loadModel(parent_folder,jsonData);		
```

Comment:
this doesn't need to be two `.then` can instead do
```js
fetch(this.data.volumeData)
  .then(res => {
    this.loadModel(parent_folder, res.json())
  })

Reply:
Changed it!

Reply:
Rolling it back. 
```
fetch(this.data.volumeData)
  .then(res => {
    this.loadModel(parent_folder, res.json())
  })
```
@mcmcgrath13  Are you sure `res.json()` has the data in the json data? 
mapping data as `volumeProperties["season"]` returns undefined.

Reply:
it ought to, though maybe it needs to be awaited.  Though this whole thing can be made moot.  If you put the `config.json` under `src/assets` then you can just `import * as config from 'src/assets/config.json'` or maybe it's `import config from 'src/assets/config.json'` and skip the whole fetch thing.

Reply:
Let's leave it as it is right now. I'll create an issue attacking this comments